cmake/FindSEACASExodus.cmake

```diff
@@ -1,32 +0,0 @@
-#------------------------------------------------------------------------------#
```

Comment:
I would keep that as exodus will be provided by exodusii-devel on debian, which doesn't have a cmake file.